ictv/renderer/renderer.py

```diff
@@ -195,6 +195,19 @@ def __init__(self, *args, **kwargs):
                 templates[os.path.splitext(template)[0]] = {}
 
         for template in templates:
+            def f(type):
```

Comment:
Is it possible to have more intelligible names than ``f`` and `g` ?

Reply:
This is old code that I deleted in the previous PR thinking it was not useful anymore. Turns out it was 😅. However I did not named the function and it is only used in the line under it.